backends/optimized/ceed-opt-basis.c

```diff
@@ -22,8 +22,7 @@
 // NOTRANSPOSE: V_ajc = T_jb U_abc
 // TRANSPOSE:   V_ajc = T_bj U_abc
 // If Add != 0, "=" is replaced by "+="
-static int CeedTensorContract_Opt(Ceed ceed,
```

Comment:
My philosophy here was that the `ceed` is needed to raise an error if we have reason to do so.

Reply:
Restored